_topics/en/segregated-witness.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Segregated witness
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: segwit
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Soft Forks
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
```

Comment:
Again, it'd be nice to have descriptions of these links when there's more than one BIP.